runtime/gc_ctrl.c

```diff
@@ -392,16 +390,30 @@ CAMLprim value caml_runtime_variant (value unit)
 #endif
 }
 
-extern int caml_parser_trace;
```

Comment:
There's one of these knocking around in startup_nat.c which could likewise be removed